NuGet.Config

```diff
@@ -12,6 +12,7 @@
   </solution>
   <packageSources>
     <clear />
+    <add key="source-built" value="C:\Users\frsilb\Documents\git\source-build\bin/obj/x64/Release/blob-feed/packages/" />
```

Comment:
hrmm?

Reply:
Whoops. Added more files than I wanted, I'll amend the commit.

Reply:
Fixed.